sql/core/src/test/resources/sql-functions/sql-expression-schema.md

```diff
@@ -0,0 +1,337 @@
+## Summary
```

Comment:
How about making the file name more clear for a golden file? For example, `sql-functions/sql-expression-schema.md` -> `sql-functions/sql-expression-schema-golden.md`.

Reply:
I don't think it's necessary. Thanks for your help.

Reply:
One more comment: plz add the same comment with the SQLQueryTestSuite like `Automatically generated by ExpressionsSchemaSuite`
https://github.com/apache/spark/blob/master/sql/core/src/test/resources/sql-tests/results/postgreSQL/boolean.sql.out#L1

Reply:
OK. I will add it.